skeleton/webpack/webpack.config.js

```diff
@@ -279,20 +280,23 @@ module.exports = ({ production, server, extractCss, coverage, analyze, karma } =
       // @if feat.babel
       {
         test: /\.js$/i, loader: 'babel-loader', exclude: nodeModulesDir,
-        options: coverage ? { sourceMap: 'inline', plugins: ['istanbul'] } : {}
+        options: tests ? { sourceMap: 'inline', plugins: ['istanbul'] } : {}
       },
       // @endif
       // @if feat.typescript
-      { test: /\.ts$/, loader: "ts-loader", options: { reportFiles: [ srcDir+'/**/*.ts'] }, include: karma ? [srcDir, testDir] : srcDir },
+      { test: /\.ts$/, loader: "ts-loader" },
```

Comment:
I forgot the story around karma and webpack because I am no fan of both. Please test karma setup.

Reply:
All release checks worked